.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -191,10 +191,10 @@ EOF
     done
 }
 # usage:              SERIES   ROT_DIR      ROT_VERSION              BOARDS...
-add_hubris_artifacts  dogfood  staging/dev  cert-staging-dev-v1.0.0  gimlet-c psc-b sidecar-b
+add_hubris_artifacts  all      staging/dev  cert-staging-dev-v1.0.0  gimlet-c gimlet-d psc-b psc-c sidecar-b sidecar-c
```

Comment:
Should we go ahead and add `gimlet-e` too? It's available in `dvt-dock`. (We do not yet have new psc/sidecar hardware revisions.)

Reply:
Might as well. I'll update that in a bit. (In the very near-term we're going to switch from using dvt-dock to tagged-and-released Hubris artifacts, hopefully tomorrow, and I was planning on adding it then.)